openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -94,6 +97,14 @@ def setUpClass(cls):
             has_score=False,
         )
 
+        # add ungraded problem for xblock_handler test
+        cls.ungraded_problem = ItemFactory.create(
+            parent=cls.blocks_dict['vertical'],
+            category='problem',
+            display_name='problem_2',
+            has_score=False,
```

Comment:
you can't set has_score when creating a problem (it is a calculated property from the actual type of the XBlock) so this isn't affecting the problem.
the problem happens to still be ungraded because you didn't set graded=True.  
I noticed you had has_score=False on other problems, so you can remove that as well